plugins/mcreator-localization/help/default/plant/can_be_placed_on.md

```diff
@@ -0,0 +1,3 @@
+List of blocks the plant can be placed on.
+
+NOTE: Overrides the plant type.
```

Comment:
The placement conditions no longer override the plant type, so you should update the tooltips and the GUI texts. You could write that the plant type is used to check the valid position if no block to grow on/condition is added

Reply:
Okay, I'll change that.

Reply:
Passing back on it, why does the placement condition no longer override the plant type? I'll take the "grass example" again: if I don't add grass to the list of blocks the plant can be placed on, but the list isn't empty, the plant won't be able to be placed on grass even if the plant type is set to `plain`

Reply:
It's because plants need to have a plant type anyway, so there's no reason to override it. If you use the "Is plant of plant type" procedure block on this grass plant, it'll return true for `plains`, even if it can't be placed on grass

Reply:
I would change this to at least overrides plant type placement condition, as it indeed keeps the plant type parameter, just changes how it works

Reply:
This is not resolved

Reply:
Fixed.